pipelines/managed/push-rpm-to-koji/push-rpm-to-koji.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: push-rpm-to-koji
   labels:
-    app.kubernetes.io/version: "0.1.0"
+    app.kubernetes.io/version: "0.2.0"
```

Comment:
you removed the README change (thanks) but didn't revert this version string bump

Reply:
Oops, I thought I ran `git checkout` on the whole directory, but I ran it only on the README. Fixed.